great_expectations/expectations/metrics/column_aggregate_metrics/column_quantile_values.py

```diff
@@ -329,6 +340,25 @@ def _get_column_quantiles_sqlite(
         raise pe
 
 
+def _get_column_quantiles_athena(
+    column,
+    quantiles: Iterable,
+    allow_relative_error: bool,
```

Comment:
The `allow_relative_error` kwarg doesn't appear to be used

Reply:
done